src/tools/perf/api/libperf.h

```diff
@@ -210,12 +209,16 @@ typedef struct ucx_perf_params {
         char                   md_name[UCT_MD_NAME_MAX];      /* Memory domain name to use */
         uct_perf_data_layout_t data_layout; /* Data layout to use */
         unsigned               fc_window;   /* Window size for flow control <= UCX_PERF_TEST_MAX_FC_WINDOW */
+        size_t                 am_hdr_size; /* UCT Active Message header size
```

Comment:
why it can't be a common field?

Reply:
it is easier to keep the default value this way, because they are different for UCT and UCP